src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -157,8 +157,12 @@ int ACA_OVS_L2_Programmer::setup_ovs_bridges_if_need()
   } else {
     // case 3: only one of the br-int or br-tun is there,
     // Invalid environment so return an error
-    ACA_LOG_CRIT("Invalid environment br-int=%d and br-tun=%d, cannot proceed\n",
-                 br_int_existed, br_tun_existed);
+    if (br_int_existed)
+        ACA_LOG_DEBUG("Valid environment for br-int but Invalid for br_tun = %d\n", br_tun_existed);
+    else
+        ACA_LOG_DEBUG("Valid environment for br-tun but Invalid for br_int = %d\n", br_int_existed);
```

Comment:
Maybe we can change it to something the that user understands more, such as:

`You have br-x but doesn't have br-y, please create the non-existent ovs bridge or delete the existing one before you try again.`

even better, you might also tell them what command to use to delete the bridge, which is:
`sudo ovs-vsctl del-br br-x`